server/src/types.ts

```diff
@@ -374,7 +379,10 @@ export type JobItem =
 
   // OCR
   | { name: JobName.OcrQueueAll; data: IBaseJob }
-  | { name: JobName.Ocr; data: IEntityJob };
+  | { name: JobName.Ocr; data: IEntityJob }
+
+  // Workflow
+  | { name: JobName.AssetCreateWorkflow; data: IAssetCreateWorkflowJob };
```

Comment:
I would probably prefer a single job for all workflow runs, instead of one per trigger type